core/modules/parser/FromFactory.cc

```diff
@@ -489,9 +489,9 @@ class FromFactory::RefGenerator {
     ///    column_name (COMMA column_name)*
     /// ;
     std::shared_ptr<query::ColumnRef> _processColumn(RefAST sib) const {
-        if(!sib.get()) {
+        if (!sib.get()) {
             throw ParseException("NULL column node", sib); }
-        if(sib->getType() != SqlSQL2TokenTypes::REGULAR_ID) {
+        if (sib->getType() != SqlSQL2TokenTypes::REGULAR_ID) {
             throw ParseException("Bad column node for USING", sib); }
```

Comment:
well, I guess these are fairly readable if there are several in a row this way, and you can at least set breakpoints on them this way, so maybe disregard comment above?